back/engines/commercial/multi_tenancy/lib/tasks/core/fix_inconsistent_data.rake

```diff
@@ -62,44 +62,6 @@ namespace :inconsistent_data do
     end
   end
 
-  task fix_pc_that_cannot_contain_ideas: :environment do
```

Comment:
Curious why was this deleted?

Reply:
I strongly doubt that we would ever reuse this (in general we have too many rake tasks so I think it's good to try to remove some), and the occurrence of `can_contain_ideas?` meant I would have to change it, so I thought it was not worth the effort.